include/soci/use.h

```diff
@@ -48,10 +48,16 @@ struct use_container<T, no_indicator>
 
 // soci::use is deleted for rvalues because it will likely lead to subtle stack-use-after-scope bugs.
```

Comment:
This comment is not true any more and really should be updated, it's just too confusing otherwise.